src/CabanaPD_Particles.hpp

```diff
@@ -87,6 +87,7 @@ class Particles<MemorySpace, PMB, TemperatureIndependent, Dimension>
   public:
     using self_type =
         Particles<MemorySpace, PMB, TemperatureIndependent, Dimension>;
+    using thermal_type = TemperatureIndependent;
```

Comment:
I am confused here. First we pass "TemperatureIndependent" in Line 89, but then define "thermal_type"?

Should one first define thermal_type and then pass thermal_type instead in Line 89?

Same for Line 505 below.

Reply:
We need `thermal_type` externally. Which one we use here does not make any difference

Reply:
I guess I was wondering if it would be clearer to pass "thermal_type" (after being defined) instead. Otherwise, it's OK.